### DISCUSSION TITLE
How to select A

### DISCUSSION THREAD

**Discussion by Stefan553** [pschatzmann/arduino-audio-tools#511] [Q&A] [answered]
Hello I'm trying to use your arduino libary for my  ai_thinker (ES8388) 2957 3478 A149 (I think its that one). But theres one thing I cant figure out. How do I specify this board in the arduino IDE itself? I looked at the readme and the example in the video https://www.youtube.com/watch?v=a936wNgtcRA but I couldn't figure out that detail yet. 

Any help would be appreciated .
Thanks.

**Comment by pschatzmann** [comment, marked as answer]
Just read the documentation of the [AudioKit](https://github.com/pschatzmann/arduino-audiokit) project.
There is also plenty of additional information in the related Wiki

**Reply by Stefan553** [reply]
Thank you for the quick answer. 

I did the tweaking in necessary tweaking in AudioKitSettings.h like the readme descriped. And I looked at the Wiki too but couldn't find an example how to tell the programm Arduino IDE in the board manager which board I'm using. I'm not that experienced with arduino so that question might be silly. But for me it seems like arduino IDE needs to know which board I'm using? And I couldn't find any examples in the documentation or wiki of that specific detail. 

Thanks.

**Reply by pschatzmann** [reply]
I am using the generic ESP32 Dev Module

**Reply by Stefan553** [reply]
Perfect. Thanks for the answer. Now it working properly.
